.github/workflows/fuzzing.yml

```diff
@@ -12,8 +12,6 @@ on:
 
 jobs:
   fuzzing:
-    if: (github.repository == 'tarantool/tarantool')
```

Comment:
AFAIR, this workflow requires specific secrets to trigger OSS Fuzz side activities. And here we don't use our own runners, so, I guess, it'll try to start and will fail in a fork.

Reply:
Answered [here](https://github.com/tarantool/tarantool/pull/6630#discussion_r754860668).

Reply:
Okay.